app/code/Magento/WishlistGraphQl/etc/module.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
+    <module name="Magento_WishlistGraphQl" />
```

Comment:
Shouldn't it be sequence from `Magento_Wishlist` module

Reply:
we very briefly talked about that, but left it out since there is no need for it (at least so far). Not sure if it is needed at all as long as this module only implements the graphql logic, since we are not changing any configurations that would rely on the wishlist module